controllers/configmap_controller.go

```diff
@@ -579,8 +580,8 @@ func (r *ConfigMapReconciler) ensureWICDRoleBinding() error {
 	}
 	if err == nil {
 		// check if existing RoleBinding's contents are as expected, delete it if not
-		if existingRB.RoleRef == expectedRB.RoleRef &&
-			len(existingRB.Subjects) == len(expectedRB.Subjects) && existingRB.Subjects[0] == expectedRB.Subjects[0] {
+		if existingRB.RoleRef.Name == expectedRB.RoleRef.Name &&
```

Comment:
any reason to not do a deep equal on the RoleRef as well?

Reply:
Yes. Please see https://github.com/openshift/windows-machine-config-operator/pull/1671#discussion_r1246976250.